src/compilationSamples/samples/expectSaga-test.ts

```diff
@@ -53,16 +53,16 @@ const watchForPostText = forEvery(postText, async ($, { text }) => {
 test('Saga test', async () => {
   expectSaga(watchForPostText)
     .withReducer(sampleIdentityCountReducer)
-    .withMocks([select(stringLongerThanCountSelector, 5 /* should be `boolean` */)])
+    .andMocks([select(stringLongerThanCountSelector, 5 /* should be `boolean` */)])
+    .whenDispatched(wrongAction({ notText: 'asdf' }))
     .toCall(fetch, new Response(undefined, { status: 200 }))
     .toCall(fetch, 404)
```

Comment:
@HenriBeck Ah, this used to error, but now doesn't anymore.

Reply:
Yes, because the dispatch is an error now, and that silents the following errors or TS doesn't check that code in that case.